docs/SECURITY.md

```diff
@@ -0,0 +1,9 @@
+# Reporting Security Issues
+
+The lichess-bot team and community take security bugs in lichess-bot seriously. We appreciate your efforts to responsibly disclose your findings, and will make every effort to acknowledge your contributions.
+
+To report a security issue, please use the GitHub Security Advisory ["Report a Vulnerability"](https://github.com/lichess-bot-devs/lichess-bot/security/advisories/new) tab.
+
+The lichess-bot team will send a response indicating the next steps in handling your report. After the initial reply to your report, the security team will keep you informed of the progress towards a fix and full announcement, and may ask for additional information or guidance.
```

Comment:
In the second sentence, change "the security team" to "the team", since we don't have a separate security team.